core/src/main/resources/i18n/displayStrings_de.properties

```diff
@@ -848,6 +848,7 @@ funds.withdrawal.inputs=Eingaben Auswahl
 funds.withdrawal.useAllInputs=Alle verfügbaren Eingaben benutzen
 funds.withdrawal.useCustomInputs=Spezifische Eingaben benutzen
 funds.withdrawal.receiverAmount=Empfängers Betrag
+funds.withdrawal.sendMax=
```

Comment:
Any incomplete translations should be omitted completely, so it falls back to english.

Reply:
removed